### PR TITLE
added unit test

### PR DESCRIPTION
**Pull request by olaoyefaith**

### CODE REVIEW

**Comment by t-bello7** [comment]
Nice one @olaoyefaith But for some reason the workflow was not picked up by Github actions.

File: .github/workflows/unit_test_ci.yaml [hunk 1]
```diff
@@ -0,0 +1,53 @@
+name: unit test ci
+
+on:
+  push:
+    branches:
+      - '*'
+
```
Comment (t-bello7) [inline .github/workflows/unit_test_ci.yaml:7]: The workflow is not picked up by Github actions because there is no trigger for a pull request.

**Review by t-bello7 — CHANGES_REQUESTED** [review]

File: .github/workflows/unit_test_ci.yaml (hunk 1 above)
Comment (olaoyefaith) [reply .github/workflows/unit_test_ci.yaml:7]: alright will add it now thanks

**Review by Fbayomide — APPROVED** [review]
You've added the pull request trigger to the workflow. Well done, Faith.

**Pull request merged**